docs/unified-test-documentation/dasharo-compatibility/322-docking-station-audio.md

```diff
@@ -0,0 +1,259 @@
+# Dasharo Compatibility: Docking station audio
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
+1. The docking station connected to the USB-C port.
+
+### DAU001.001 Audio recognition (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that the external headset is properly recognized
+after plugging in the 3.5 mm jack into the docking station.
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Plug in a headset jack into the docking station.
+1. When the `Select Audio Device` menu appears, select what type of external
+    device has been connected to the laptop (headset).
+1. Open a terminal window and execute the following command:
+
+```bash
+amixer -c 0 contents | grep -A 2 'TBD'
```

Comment:
`TBD` cannot be left on the documentation. We should improve this point.

Reply:
replaced